src/serialization/src/main/kotlin/org/vaccineimpact/api/serialization/Deserializer.kt

```diff
@@ -32,7 +31,7 @@ class Deserializer
         {
             String::class.createType() -> raw
             Int::class.createType() -> raw.toInt()
-            BigDecimal::class.createType() -> BigDecimal(raw)
+            Float::class.createType() -> raw.toFloat()
```

Comment:
Don't we want to be able to handle both of these types? So just add the float code, but no need to remove the decimal.

Reply:
So I wondered the same about removing the ability to parse decimals, but I think we should for now, given none of our target types at the moment have decimals in them (and maybe now none of our database data structures?) - if we need to parse decimals again down the line, we can add it back in.

Reply:
We have decimals in the db - coverage and demographics - but it's true that the API doesn't write to those at the moment.